pkg/solutions/base.go

```diff
@@ -92,3 +94,37 @@ func (w *TextWriter) WriteFilter(out io.Writer, sols []Solution, filter Filter)
 func NewTextWriter() *TextWriter {
 	return &TextWriter{}
 }
+
+type Query struct {
+	owner           string
```

Comment:
Since this is the generic `Query` object, `owner` as a variable name seems out of place here. Consider something perhaps as equally as generic as `Query`; `value` or `val` or something might even be just fine.

Reply:
@nathanagood So what I was thinking here was if we add more queries in the future (I.E tags, version, etc) we could add more fields to the struct. With another function like this https://github.com/cloud-native-toolkit/itzcli/pull/14/files/38630b5968e235f2319819da776e6064a1617930#diff-fe88464b4431bd2c48178a3753976f9097fc13ea52fc5f70739005594f7d1c6eR104 that would update the fields depending on what is supplied, and the BuildQuery would handle grabbing the fields and concatenating them. 

So like a TagQuery function would update the field tag.